### PR TITLE
Add groovy files

### PR DESCRIPTION
**Pull request by ESI-SYD**
This PR include 3 groovy files

`inductor_dashboard.groovy`: For op microbench and future model bench
`inductor_image.groovy`:For docker images build and push
`inductor_graph.groovy`:cosim tool automation job used for graph dump

### CODE REVIEW

**Comment by ESI-SYD** [comment]
@chuanqi129 Please have a time to review, thanks~

File: groovy/inductor_graph.groovy [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+        # set -x
+        SUITE=$1
+        MODEL=$2
+        CHANNELS=$3
+        BS=$4
+        TARGET_TAG=$5
+
+        MODEL_LIST=($(echo "${MODEL}" |sed '"'"'s/,/ /g'"'"'))
+
+        for SINGLE_MODEL in ${MODEL_LIST[@]}
```
Comment (chuanqi129) [inline groovy/inductor_graph.groovy:42]: Can we move out the `for` loop, and do the `for` loop for `docker exec ...`? I feel this method is more readable. You can decide which is a better method

File: groovy/inductor_graph.groovy [hunk 2]
```diff
@@ -0,0 +1,119 @@
...
+        # set -x
+        SUITE=$1
+        MODEL=$2
+        CHANNELS=$3
+        BS=$4
+        REFERENCE_TAG=$5
+
+        MODEL_LIST=($(echo "${MODEL}" |sed '"'"'s/,/ /g'"'"'))
+
+        for SINGLE_MODEL in ${MODEL_LIST[@]}
```
Comment (chuanqi129) [inline groovy/inductor_graph.groovy:77]: Same here

File: groovy/inductor_dashboard.groovy [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+                docker stop $old_container
+                docker rm $old_container
+                docker container prune -f
+            fi
+            old_image_id=`docker images|grep pt_inductor|grep nightly|awk '{print $3}'`
+            old_image=`echo $old_image_id| awk '{print $1}'`
+            if [ -n "${old_image}" ]; then
+                docker rmi -f $old_image
+            fi 
+            cd ${WORKSPACE}
```
Comment (chuanqi129) [inline groovy/inductor_dashboard.groovy:36]: for the workspace clean up, you can call `deleteDir()` directly

File: groovy/inductor_images.groovy [hunk 4]
```diff
@@ -0,0 +1,54 @@
+NODE_LABEL = 'images_build_inductor_dashboard'
+if ('NODE_LABEL' in params) {
+    echo "NODE_LABEL in params"
+    if (params.NODE_LABEL != '') {
+        NODE_LABEL = params.NODE_LABEL
+    }
+}
+echo "NODE_LABEL: $NODE_LABEL"
+
```
Comment (chuanqi129) [inline groovy/inductor_images.groovy:9]: Same here for parameters

**Comment by chuanqi129** [comment]
@ESI-SYD we'd better to separate the python scripts and groovy scripts into 2 PRs to make the commits clear, and the python scripts changes can be merged firstly.

File: groovy/inductor_dashboard.groovy [hunk 5]
```diff
@@ -0,0 +1,361 @@
...
+                [$class: 'StringParameterValue', name: 'TORCH_TEXT_COMMIT', value: "${TORCH_TEXT_COMMIT}"],
+                [$class: 'StringParameterValue', name: 'TORCH_DATA_BRANCH', value: "${TORCH_DATA_BRANCH}"],
+                [$class: 'StringParameterValue', name: 'TORCH_DATA_COMMIT', value: "${TORCH_DATA_COMMIT}"],
+                [$class: 'StringParameterValue', name: 'TORCH_AUDIO_BRANCH', value: "${TORCH_AUDIO_BRANCH}"],
+                [$class: 'StringParameterValue', name: 'TORCH_AUDIO_COMMIT', value: "${TORCH_AUDIO_COMMIT}"],
+                [$class: 'StringParameterValue', name: 'TORCH_BENCH_BRANCH', value: "${TORCH_BENCH_BRANCH}"],
+                [$class: 'StringParameterValue', name: 'TORCH_BENCH_COMMIT', value: "${TORCH_BENCH_COMMIT}"],
+                [$class: 'StringParameterValue', name: 'BENCH_COMMIT', value: "${BENCH_COMMIT}"],
+                [$class: 'StringParameterValue', name: 'inductor_tools_branch', value: "${inductor_tools_branch}"],
+                [$class: 'StringParameterValue', name: 'tag', value: 'nightly'],
```
Comment (chuanqi129) [inline groovy/inductor_dashboard.groovy:247]: We don't need hard code for the image `tag`. Also use a parameter for it with "nightly" default value

**Pull request closed without merging**